CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -418,10 +418,9 @@ def validate_workbook(file, args=None):
             if ws.title == configurationSheet:
                 wbMissingSheets = checkConfigurationSheet(wb, ws, configurationSheetColumnName, wsOut, verbose)
         except Exception as e:
-            print("FATAL ERROR in worksheet %s : %s" % (ws.title, str(e)))
             if debugMode:
                 tb.print_exc(e)
-            exit(-1)
+                raise FatalError("FATAL ERROR in worksheet %s : %s" % (ws.title, str(e)))
```

Comment:
This changes the behavior since this is only raised in debugmode

Reply:
Thanks. Wrong indentation there.